setup.py

```diff
@@ -120,6 +120,7 @@ def get_outputs(self):
 
 setup(
     name='dumb-init',
+    license='MIT License',
```

Comment:
I believe the value for this is supposed to be just "MIT"

it may makes sense to feed this through https://github.com/asottile/setup-py-upgrade https://github.com/asottile/setup-cfg-fmt 

these will add the rest of the license metadata automatically

Reply:
I tried running `setup-py-upgrade` on this repo but it fails with the following exception:
```
Traceback (most recent call last):
  File "/z/home/seoj/.local/lib/python3.6/site-packages/setup_py_upgrade.py", line 125, in visit_Call
    value = ast.literal_eval(kwd.value)
  File "/usr/lib/python3.6/ast.py", line 85, in literal_eval
    return _convert(node_or_string)
  File "/usr/lib/python3.6/ast.py", line 84, in _convert
    raise ValueError('malformed node or string: ' + repr(node))
ValueError: malformed node or string: <_ast.Call object at 0x7fa99e599898>
```
Do you know why?

Reply:
> I believe the value for this is supposed to be just "MIT"
You're right. [License definition](https://opensource.org/licenses/MIT)

Reply:
ah yeah the upgrade script tries its best but there's a bunch of constructs it simply cannot handle -- in this case I believe it's crashing on `version` (since it doesn't quite know how to properly convert that to `file:`)

if you temporarily remove those from the `setup(...)` call it'll do the rest of the things

Reply:
Getting this error after deleting the `version` line:

```
seoj@edderfugl:git$ setup-py-upgrade dumb-init/
c_executables= is not supported in setup.cfg
```

Reply:
this applies from master:

```diff
$ git diff origin/master 
diff --git a/setup.cfg b/setup.cfg
new file mode 100644
index 0000000..99121fe
--- /dev/null
+++ b/setup.cfg
@@ -0,0 +1,23 @@
+[metadata]
+name = dumb_init
+version = file: VERSION
+description = Simple wrapper script which proxies signals to a child
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/Yelp/dumb-init/
+author = Yelp
+license = MIT
+license_file = LICENSE
+platforms = linux
+classifiers =
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+
+[options]
+python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
diff --git a/setup.py b/setup.py
index 0ec2f18..aaa4d44 100644
--- a/setup.py
+++ b/setup.py
@@ -119,12 +119,6 @@ class build_cexe(Command):
 
 
 setup(
-    name='dumb-init',
-    description='Simple wrapper script which proxies signals to a child',
-    version=open('VERSION').read().strip(),
-    author='Yelp',
-    url='https://github.com/Yelp/dumb-init/',
-    platforms='linux',
     c_executables=[Extension('dumb-init', ['dumb-init.c'])],
     cmdclass={
(venv) asottile@asottile-MacBookPro:/tmp/dumb-init
$ git diff origin/master  | cat
diff --git a/setup.cfg b/setup.cfg
new file mode 100644
index 0000000..99121fe
--- /dev/null
+++ b/setup.cfg
@@ -0,0 +1,23 @@
+[metadata]
+name = dumb_init
+version = file: VERSION
+description = Simple wrapper script which proxies signals to a child
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/Yelp/dumb-init/
+author = Yelp
+license = MIT
+license_file = LICENSE
+platforms = linux
+classifiers =
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+
+[options]
+python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
diff --git a/setup.py b/setup.py
index 0ec2f18..aaa4d44 100644
--- a/setup.py
+++ b/setup.py
@@ -119,12 +119,6 @@ class build_cexe(Command):
 
 
 setup(
-    name='dumb-init',
-    description='Simple wrapper script which proxies signals to a child',
-    version=open('VERSION').read().strip(),
-    author='Yelp',
-    url='https://github.com/Yelp/dumb-init/',
-    platforms='linux',
     c_executables=[Extension('dumb-init', ['dumb-init.c'])],
     cmdclass={
         'bdist_wheel': bdist_wheel,
```